src/components/ebay-button/template.marko

```diff
@@ -9,4 +8,10 @@
     disabled=data.disabled
     aria-disabled=data.partiallyDisabled
     ${data.htmlAttributes}>
+    <if(data.label)>
+        ${data.label}
+    </if>
+    <else>
+        <span w-body body-only-if(true)/>
```

Comment:
So, it would always add an inner `<span>`?

Reply:
Exactly. This way we render `label` if it exists, and otherwise fall back to `w-body` (span is not rendered).